api/src/main/java/io/opencensus/stats/Aggregation.java

```diff
@@ -221,7 +220,7 @@ public static Mean create() {
      * @since 0.8
      */
     public static Distribution create(BucketBoundaries bucketBoundaries) {
-      checkNotNull(bucketBoundaries, "bucketBoundaries should not be null.");
+      Utils.checkNotNull(bucketBoundaries, "bucketBoundaries should not be null.");
```

Comment:
Maybe we can use static import to minimize all these "Utils.xxxx" changes?

Reply:
I'd rather qualify these methods to distinguish them from the methods in Preconditions, especially since we still use Guava in other parts of the project.

Reply:
I think we all agree to no longer use Guava, so I think it is fine to use static imports.

Reply:
Do you mean that you also want to remove the Guava dependencies from the implementation and exporters?  I think that should be a separate change.  I haven't looked at how we use Guava in the other artifacts, but if we use it heavily, it could be better to shade it than remove it.